core-services/prow/02_config/generate-boskos.py

```diff
@@ -73,6 +73,12 @@
     'aws-perf-qe-quota-slice': {
         'us-west-2': 3,
     },
+    'aws-perfscale-qe-quota-slice': {
+        'us-west-2': 3,
+    },
+    'aws-perfscale-lrc-qe-quota-slice': {
+        'us-west-2': 3,
```

Comment:
This one we don't need as much but slightly more. 
Can we set it to:
'us-west-2': 5

Reply:
Yeah, even when using `aws-qe` I have hit lease issues and scaling issues for Rosa and HCP clusters.

Reply:
setting `aws-perfscale-lrc-qe-quota-slice` to 5, @qiliRedHat is it OK for lrc account?

Reply:
@yunjiang29 I'm ok with 5 for lrc account.